src/Calculator/Resources/en-US/Resources.resw

```diff
@@ -3379,4 +3379,12 @@
     <value>Microsoft Services Agreement</value>
     <comment>Displayed on a link to the Microsoft Services Agreement in the about this app information</comment>
   </data>
-</root>
+  <data name="UnitAbbreviation_Pyeong" xml:space="preserve">
+    <value>Pyeong</value>
```

Comment:
Wikipedia lists the abbreviation as `py`.  Are you sure we should use `Pyeong` explicitly?  Americans would recognize `ft` as `feet`.  Would Koreans recognize `py` as `Pyeong`?

Reply:
Hmm, good question.
I am not sure if Koreans would recognize `py` mainly because they would just write it in Korean ( 평 ).
I don't feel strongly one way or the other though.
Would you like me to update it to `py`?

Reply:
If we're uncertain, let's go with the full name.